plugins/cros-ec/fu-cros-ec-firmware.c

```diff
@@ -13,13 +13,52 @@
 #define MAXSECTIONS	2
 
 struct _FuCrosEcFirmware {
-	FuFmapFirmware			parent_instance;
-	struct cros_ec_version		version;
-	FuCrosEcFirmwareSection		sections[MAXSECTIONS];
+	FuFmapFirmware			 parent_instance;
+	struct cros_ec_version		 version;
+	GArray				*sections;
```

Comment:
I think a GPtrArray might be more useful here, no?

Reply:
Done.